display_list/dl_color.h

```diff
@@ -9,10 +9,32 @@
 
 namespace flutter {
 
+enum class DlColorSpace { kSRGB = 0, kExtendedSRGB = 1, kDisplayP3 = 2 };
+
 struct DlColor {
  public:
-  constexpr DlColor() : argb_(0xFF000000) {}
-  constexpr explicit DlColor(uint32_t argb) : argb_(argb) {}
+  constexpr DlColor()
+      : alpha_(0.f),
+        red_(0.f),
+        green_(0.f),
+        blue_(0.f),
+        color_space_(DlColorSpace::kSRGB) {}
+  constexpr explicit DlColor(uint32_t argb)
+      : alpha_(((argb >> 24) & 0xff) / 255.f),
```

Comment:
There are ToF and ToC static functions in the private part of this class to standardize on the conversions.

Reply:
done